src/features/BroadcastTransaction/stateFactory.tsx

```diff
@@ -70,7 +70,10 @@ const BroadcastTxConfigFactory: TUseStateReducerFactory<State> = ({ state, setSt
       cb();
     } catch (err) {
       console.debug(`[BroadcastTx] ${err}`);
-      displayToast(toastTemplates.failedTransaction);
+      setState((prevState: State) => ({
+        ...prevState,
+        error: err.message
+      }));
```

Comment:
We need to add in some handling for whether the tx went through or not, so that we can display to the user the tx succeeded even if we failed to process the result & display a txReceipt.

Reply:
Hmm, unsure of how that would work. In general if it succeeds we should be able to show a receipt. Probably out of scope for this anyway.